xgboost-integration.ipynb

```diff
@@ -0,0 +1,279 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# XGBoost integration\n",
+    "Here, we present XGBoost integration with Neptune that lets you automatically log metrics (train, eval), save trained model to Neptune and much more.\n",
+    "\n",
+    "Usage is easy: just pass `neptune_callback` to training function like any other xgboost callback.\n",
+    "\n",
+    "To try integration simply run this Notebook top to bottom. It works outside-the-box :)\n",
+    "\n",
+    "You can log multiple data types:\n",
+    "\n",
+    "    * Log metrics (train and eval) after each boosting iteration.\n",
+    "    * Log model (Booster) to Neptune after last boosting iteration.\n",
+    "    * Log feature importance to Neptune as image after last boosting iteration.\n",
+    "    * Log visualized trees to Neptune as images after last boosting iteration."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Resources\n",
+    "* Use case / tutorial: https://docs.neptune.ai/integrations/xgboost.html,\n",
+    "* Open source implementation is on [GitHub](https://github.com/neptune-ai/neptune-contrib/blob/master/neptunecontrib/monitoring/xgboost_monitor.py),\n",
+    "* Docstrings / reference documentation is [here](https://neptune-contrib.readthedocs.io/user_guide/monitoring/xgboost.html)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visual overview\n",
+    "![xgboost-integration-tour](./_static/xgboost-tour.gif \"XGBoost integration tour\")"
```

Comment:
This doesn't work on google colab. You need to put it as URL. 
What I usually do is just put it on our website in the media upload and use that link. 
You can create gist as well if you want to but either way it has to be a URL,

Reply:
it's fixed now.